bip-0016.mediawiki

```diff
@@ -116,4 +116,4 @@ https://gist.github.com/gavinandresen/3966071
 * [[bip-0016/qa.mediawiki|Quality Assurance test checklist]]
 
 == References ==
-<references>
+<references></references>
```

Comment:
If there is nothing enclosed, it would be preferable to use a self-closing tag:

```suggestion
<references/>
```

Reply:
@murchandamus updated with your suggestion here in #1998